### PR TITLE
JNG-4533 Release osgi-asciidoc-converter

### PR DESCRIPTION
**Pull request by gaborflorian**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4533" title="JNG-4533" target="_blank">Sub-taskJNG-4533</a>  Release osgi-asciidoc-converter
  </td></table>
  <br />

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -117,6 +131,18 @@
...
+
+                <plugin>
+                    <groupId>org.sonarsource.scanner.maven</groupId>
+                    <artifactId>sonar-maven-plugin</artifactId>
+                    <version>${sonar-maven-plugin-version}</version>
+                </plugin>
+
                 <plugin>
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>3.0.0-M5</version>
```
Comment (robertcsakany) [inline pom.xml]: Incosinsten version - M5? When pluginManagement defines a plugin version is not required in plugins section

**Review by robertcsakany — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (gaborflorian) [reply pom.xml]: Unfortunetly throw error with the new M7 version

**Review by bencegelei — APPROVED** [review]

**Review by bencegelei — CHANGES_REQUESTED** [review]
surefire version inconsistency (M5 vs M7)

File: pom.xml (hunk 1 above)
Comment (gaborflorian) [reply pom.xml]: [8359bb6](https://github.com/BlackBeltTechnology/osgi-asciidoc-converter/pull/4/commits/8359bb6bcd7ba075217aecf5d314ebc6669de4d1)

**Review by bencegelei — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -30,6 +30,20 @@
         <asciidoctorj-version>2.5.2</asciidoctorj-version>
         <jruby-version>9.2.19.0</jruby-version>
 
+        <surefire-version>3.0.0-M5</surefire-version>
```
Comment (noherczeg) [inline pom.xml]: Should be M7

**Review by noherczeg — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (gaborflorian) [reply pom.xml]: Api incompatibility error with M7

**Review by robertcsakany — CHANGES_REQUESTED** [review]
Please use M7 version of surefire plugni.

File: pom.xml (hunk 2 above)
Comment (gaborflorian) [reply pom.xml]: [08500da](https://github.com/BlackBeltTechnology/osgi-asciidoc-converter/pull/4/commits/08500da679cd5dfed2c00fea990110351f4bdf58)

**Pull request merged**